automated-comments/elixir/two-fer/use_specification.md

```diff
@@ -0,0 +1,16 @@
+<!--
+This warning is raised when the solution doesn't have a type sepcification
```

Comment:
Again, shouldn't this be a generic comment?

Reply:
moved to solution rather that two-fer